src/bulmaOverride.css

```diff
@@ -93,3 +93,17 @@ h6 {
 ion-icon {
   pointer-events: none;
 }
+
+.note {
+  display: block;
+  font-size: 0.75rem;
+  background-color: whitesmoke;
+  border-left: 4px #dbdbdb solid;
+}
+
+.note a {
+  text-decoration: underline dotted;
```

Comment:
I'm really not liking how this looks. To me it makes the text of the note difficult to read. I would prefer just using the hover style `text-decoration-style: underline solid;` for both states.

Reply:
I do think there should be some change when hovered over. I find the current instruction links not obviously links, which is why I changed it here.

Reply:
That's fair. It's already doing a color and cursor change on hover which is enough for me. If you think we need more how about making these look like buttons that are more obviously clickable. Using a button would make it slightly more apparent that these instructions links pull up the same modal as the big instructions button below the content iframe.

Reply:
So, for me, the color change is very subtle; I didn't intentionally added and did not notice it when I originally wrote this. I wonder if getting rid of the color (adding `color: unset` to this block) would address the readability issues.

Reply:
Changed the style to
```
.note a {
  color: unset;
  text-underline-offset: 2px;
  text-decoration: underline dotted;
}
```
Chromium
![image](https://user-images.githubusercontent.com/4044095/225011588-f69fb4f2-fc41-436c-a686-3207df0b63a4.png)

Firefox
![image](https://user-images.githubusercontent.com/4044095/225011951-a1d661c7-93ea-45af-8012-93faf65216cf.png)